saleor/webhook/tests/test_webhook_payloads.py

```diff
@@ -372,3 +373,21 @@ def test_generate_payment_payload(dummy_webhook_app_payment_data):
         dummy_webhook_app_payment_data.gateway
     ).name
     assert payload == json.dumps(expected_payload, cls=CustomJsonEncoder)
+
+
+def test_generate_customer_payload_with_proper_shipping_and_billing_address(
```

Comment:
Instead of adding specific tests for `proper_shipping_and_billing_address`, you can just add a normal test for `generate_customer_payload` as it is missing.